flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/FlinkRelNode.scala

```diff
@@ -32,6 +34,18 @@ import scala.collection.JavaConversions._
   */
 trait FlinkRelNode extends RelNode {
 
+  /**
+    * Returns the display name of the RelNode. The display name are usually used to be displayed
+    * in log or Web UI.
+    */
+  def getDisplayName: String = {
+    val sw = new StringWriter
+    val pw = new PrintWriter(sw)
+    val relWriter = new RelDisplayNameWriterImpl(pw)
```

Comment:
The `display name` is the concept of `Operator`, not `RelNode`.  `OperatorNameGenerator` is better ?